auth/src/main/res/layout/fui_welcome_back_idp_prompt_layout.xml

```diff
@@ -1,29 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<ScrollView
```

Comment:
Same question here, this is a relatively "short" view that does not seem in danger of scrolling.

Reply:
@SUPERCILEX have you noticed a problem with partially visible layout on any device? For me, it would be only reason to keep here `ScrollView`. If no, please remove it as it would decrease the app performance and increase app building process.